Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1111,12 +1111,20 @@ class CombinatorialKalmanFilter {
                        firstCreatedState.smoothedCovariance(),
                        firstCreatedState.referenceSurface());
         reverseDirection = (firstIntersection.intersection.pathLength < 0);
+        // Reinitialize the stepping jacobian
+        state.stepping.jacToGlobal =
```

Comment:
After a second thought, I'm not sure if we really need to reset the `jabToGlobal` after calling the stepper `update` as the `jacToGlobal` is already reset there.

Reply:
Your right, it is already reset in the `update` I will remove it